src/video/x11/SDL_x11keyboard.c

```diff
@@ -841,6 +842,45 @@ X11_SetTextInputRect(_THIS, const SDL_Rect *rect)
 #endif
 }
 
+SDL_bool
+X11_HasScreenKeyboardSupport(_THIS)
+{
+    SDL_VideoData *videodata = (SDL_VideoData *) _this->driverdata;
+    return videodata->is_steam_deck;
+}
+
+void
+X11_ShowScreenKeyboard(_THIS, SDL_Window *window)
+{
+    SDL_VideoData *videodata = (SDL_VideoData *) _this->driverdata;
+
+    if (videodata->is_steam_deck) {
+        /* For more documentation of the URL parameters, see the entry for
+         * ShowFloatingGamepadTextInput:
```

Comment:
You can link directly to that API function:
https://partner.steamgames.com/doc/api/ISteamUtils#ShowFloatingGamepadTextInput